pkgs/by-name/ip/ipxe/package.nix

```diff
@@ -144,7 +144,15 @@ stdenv.mkDerivation (finalAttrs: {
   meta = {
     description = "Network boot firmware";
     homepage = "https://ipxe.org/";
-    license = lib.licenses.gpl2Only;
+    license = with lib.licenses; [
+      bsd2
+      bsd3
+      gpl2Only
+      gpl2Plus # with UBDL exception.
```

Comment:
The UBDL exeption should be added to lib.license and then be used here, similar as to how llvm-exception is also treated as a regular license in nixpkgs (mainly becouse we currently don't have a better way)